src/amo/components/HomepageShelves/styles.scss

```diff
@@ -0,0 +1,6 @@
+@import '~amo/css/styles';
+
+.HomepageShelves-loading-text {
```

Comment:
I'm surprised you need this. I would have thought it would have "just worked", as it works on the current homepage.

Reply:
That was an oversight, sorry! It does work without the import.

Reply:
I'm not sure what you mean by "the import".

Reply:
Ohhh, I thought you meant that I didn't need `@import '~amo/css/styles';` for it to work. I did not realize that I didn't need the stylesheet at all; I'll delete the stylesheet. 🤦🏻‍♀️